src/tuf_on_a_plane/config.py

```diff
@@ -27,13 +28,13 @@ class Config:
 
     # Maximum number of root rotations.
     MAX_ROOT_ROTATIONS = 2 ** 5
-    # 16K ought to be more than enough for everyone ;)
-    MAX_ROOT_LENGTH: Length = Length(2 ** 14)
+    MAX_ROOT_LENGTH: Length = Length(2 ** 15)
```

Comment:
Now that we check the upper bound on all downloads, there is an issue I don't understand right now where root and timestamp were somehow still slightly bigger than we expected. Will look into this later.

Reply:
Fixed with 2f1e818d4c28662ba971344fe94a2fdb2883b9c1